.github/workflows/release-vsphere-template.yaml

```diff
@@ -28,9 +28,9 @@ jobs:
           - os: "rocky 9.1"
             buildConfig: "offline"
           - os: "oracle 9.4"
-            buildConfig: "basic"
+            buildConfig: "offline"
```

Comment:
we should add `offline` and `offline-fips` in addition to `basic` and `fips`. we need to test for all 4 buildconfigs

Reply:
thanks, I have done it as per @dkoshkin suggestion as we're doing the same for other OS images(only offline* modes are tested) - https://github.com/mesosphere/konvoy-image-builder/pull/1192#issuecomment-2392004946 .. lmk if we want to run for all 4 buildconfigs.

Reply:
We don't have all 4 for the other OSs, but we could start here

Reply:
done f521da9876a38f6d214b3a77066bb597ff6894d3

Reply:
Thanks. I would prefer atleast one flavor of `basic` tests. Sometimes we do miss failure with `networked` tests.